src/staging/src/templates/staticContent1.js

```diff
@@ -52,43 +56,182 @@ export default function StaticContent1({ pageContext }) {
     }
   }`)
 
-  const zonesContent = pageContext?.page?.Content?.filter(c => Boolean(c.strapi_component === 'parks.html-area'))
+
+  const pageContent = pageContext?.page?.Content; // array of content components in page
   const meta = pageContext?.page?.Content.find(c => Boolean(c.strapi_component === 'parks.seo')) || {}
-  const menuContent = queryData?.allStrapiMenus?.nodes || []
+  const menuContent = queryData?.allStrapiMenus?.nodes || [] // megaMenu
+
+  // look for PageHeader content
+  // if it exists, will affect the layout of the top of the page
+  // note that it does not matter what position the component is in, it will appear at the top
+  // note that if there are more than one such component, it will pick the first
+  const headerContent = pageContext?.page?.Content.find(c => Boolean(c.strapi_component === 'parks.page-header')) || {}
+  const hasPageHeader = (headerContent.PageTitle !== undefined)
+
+  // Get page title from record
+  // if not there, get from page title, if there is a PageHeader compopnent
+  // otherwise, page title & breadcrumb assumed to be in the content
+  let pageTitle = pageContext.page.title
+  if (!pageTitle) {
+    pageTitle = headerContent.PageTitle
+  }
+  const hasTitle = (pageTitle !== undefined)
+
+  const sections = pageContent.filter(c => Boolean(c.strapi_component === 'parks.page-section')) || []
+  const hasSections = (sections.length > 0)
+
+  // create page sections for sticky sidebar menu
+  // and scrollspy highlighting
+
+  let sectionRefs = [
+    // Creating 10 refs for scrollspy
+    // TODO create dynamically without causing error
+    // these are created whether or not there are sections
+    // as useRef cannot be used conditionally
```

Comment:
Looked into this a bit and there doesn't seem to be an easy solution, fine to go with this approach for now